### ISSUE TITLE
reqwest crate's default "time out" definition should be removed

### ISSUE THREAD

**Issue by IdemenB** [Dansvidania/rust-camunda-client#3]
Hi @Dansvidania , here is an improvement for the library. As we discussed in https://github.com/Dansvidania/camunda-worker/issues/11 Camunda REST API allows long polling. For the duration you set while you perform "fetch and lock" for an external task, you can set a timer up to 30 minutes and for that long the HTTP connection is kept alive and Camunda pushes the client newly created tasks immediately thereby eliminating the constant need for making new HTTP requests to check if there is a new task.

However, reqwest crate which is used here comes with a default time out period of 30 seconds. This breaks the long polling set longer than 30 seconds. There is a configuration function to disable it.

https://docs.rs/reqwest/0.8.4/reqwest/struct.ClientBuilder.html

```
fn timeout<T>(&mut self, timeout: T) -> &mut ClientBuilder 
where
    T: Into<Option<Duration>>, 
[src][−]
Set a timeout for connect, read and write operations of a Client.

Default is 30 seconds.

Pass None to disable timeout.

```

**Comment by dan-amoroso** [comment]
makes sense. I will try to fix this asap

**Comment by dan-amoroso** [comment]
this should be fixed in 0.1.2

**Comment by IdemenB** [comment]
Tested and confirmed that the default time out is not enforced anymore.
